core/src/create.jl

```diff
@@ -202,12 +202,18 @@ function Connectivity(db::DB)::Connectivity
     edge_ids_flow_inv = Dictionary(values(edge_ids_flow), keys(edge_ids_flow))
 
     flow = adjacency_matrix(graph_flow, Float64)
+
+    if config.solver.autodiff
+        flowd = DiffCache(flow)
+        flow = get_tmp(flowd, flow)
+    end
+
     nonzeros(flow) .= 0.0
 
     return Connectivity(
         graph_flow,
         graph_control,
-        flow,
+        config.solver.autodiff ? flowd : flow,
```

Comment:
`flow` is used in line 211; that operation cannot be performed on `flowd`. Line 208 might be unnecessary, I'm not sure.

Reply:
Right, I missed that. Haven't tried, but based on my understanding of `get_tmp` line 208 is indeed not needed, since the input and output `flow` should be the same.

Reply:
It works fine without line 208